metals/src/main/scala/scala/meta/internal/metals/Doctor.scala

```diff
@@ -362,63 +438,77 @@ final class Doctor(
   private def extractTargetInfo(
       targetId: BuildTargetIdentifier
   ): List[DoctorTargetInfo] = {
-    val scalaDoctorInfo =
-      buildTargets.scalaTarget(targetId).map(extractScalaTargetInfo).toList
-    val javaDoctorInfo =
-      buildTargets.javaTarget(targetId).map(extractJavaTargetInfo).toList
-    scalaDoctorInfo ::: javaDoctorInfo
+    val javaTarget = buildTargets.javaTarget(targetId)
+    buildTargets
+      .scalaTarget(targetId)
```

Comment:
I don't think this will return anything if the JavaTarget exists and the ScalaTarget doesn't which is possible in Bloop/Gradle projects.

Reply:
So in that case we should have a separate Java target I guess, good catch!